x/order/keeper/keeper.go

```diff
@@ -323,6 +324,11 @@ func (k Keeper) GetUpdatedOrderIDs() []string {
 	return k.cache.getUpdatedOrderIDs()
 }
 
+// GetTxHandlerMsgResult: be careful, only call by backend module, other module should never use it!
+func (k Keeper) GetTxHandlerMsgResult() []bitset.BitSet {
+	return k.cache.toggleCopyTxHandlerMsgResult()
```

Comment:
add following code in GetTxHandlerMsgResult?
if !k.enableBackend {
   return nil
}

Reply:
Yes, it will be more clearly to user who want to call this function.